tools/ci-build/runtime-versioner/src/audit.rs

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use crate::{
+    command::CommandExt,
+    index::CratesIndex,
+    repo::Repo,
+    tag::{previous_release_tag, release_tags},
+    util::utf8_path_buf,
+    Audit,
+};
+use anyhow::{anyhow, bail, Context, Result};
+use camino::{Utf8Path, Utf8PathBuf};
+use smithy_rs_tool_common::release_tag::ReleaseTag;
+use std::{
+    collections::{BTreeMap, BTreeSet},
+    fs,
+    process::Command,
+};
+
+pub fn audit(args: Audit) -> Result<()> {
```

Comment:
how does this scheme play with the sdk-versioner?

Reply:
The runtime crate versions are set as part of the copy of runtime crates into the SDK (in the SDK's build.gradle.kts), so sdk-versioner doesn't touch those. It does, however, read them and update other crates to use those versions.

Similarly, the smithy-rs release sets the runtime crate versions as part of the copy in its build.gradle.kts.

I just tried setting aws-smithy-eventstream to 0.70.0 and generating a SDK and a smithy-rs release. The SDK generated and compiled successfully, but the smithy-rs release broke due to an overzealous validation in the fix-manifests tool. I'll work on correcting that.